.circleci/config.yml

```diff
@@ -28,3 +28,11 @@ jobs:
                           arguments: '--recursive'
                           from: './content/'
                           to: 's3://${AWS_BUCKET}/${S3_BUCKET_PATH}/'
+
+workflows:
+      everything:
+          jobs:
```

Comment:
Super-minor recommendation: if you bump line 34 and everything inside (below) it to the right by two spaces, it'll be more consistent with the indentation of the rest of the file, which is 6-spaces-per-level for the first few levels.

Reply:
FWIW, I probably prefer the whole thing to be standardized on two spaces like https://github.com/dockstore/dockstore/blob/develop/.circleci/config.yml